tests/test_on_data/ophys/test_imaging_interfaces.py

```diff
@@ -903,3 +905,201 @@ def test_incorrect_folder_structure_raises(self):
+        assert "experimenter" not in nwbfile
+
+        # Device checks
+        device = metadata["Ophys"]["Device"][0]
+        assert device["name"] == "Microscope"  # Default metadata because this was not included in the source metadata
+        assert "description" not in device or device.get("description", "") == ""
+
+        # ImagingPlane checks
+        imaging_plane = metadata["Ophys"]["ImagingPlane"][0]
+        assert imaging_plane["name"] == "ImagingPlane"
+        assert (
+            imaging_plane["device"] == "Microscope"
+        )  # Default metadata because this was not included in the source metadata
+        assert (
+            imaging_plane["description"] == "The plane or volume being imaged by the microscope."
+        )  # Default metadata because this was not included in the source metadata
+
+        # Optical channel checks
+        optical_channel = imaging_plane["optical_channel"][0]
+        assert (
+            optical_channel["name"] == "channel_0"
+        )  # Default metadata because this was not included in the source metadata
+        assert (
+            optical_channel["description"] == "An optical channel of the microscope."
+        )  # Default metadata because this was not included in the source metadata
+
+        # OnePhotonSeries checks
+        ops = metadata["Ophys"]["OnePhotonSeries"][0]
+        assert ops["name"] == "OnePhotonSeries"
+        assert (
+            ops["description"] == "Imaging data from one-photon excitation microscopy."
+        )  # Default metadata because this was not included in the source metadata
+        assert ops["unit"] == "n.a."  # Default metadata because this was not included in the source metadata
+        assert ops["imaging_plane"] == "ImagingPlane"
+        assert ops["dimension"] == [128, 128]
+
+
+@skip_on_python_313
+@skip_on_darwin_arm64
+class TestInscopixImagingInterfaceMovieLongerThan3Min(ImagingExtractorInterfaceTestMixin):
+    """Test InscopixImagingInterface with movie_longer_than_3_min.isxd (rich metadata file)."""
+
+    data_interface_cls = InscopixImagingInterface
+    save_directory = OUTPUT_PATH
+    interface_kwargs = dict(
+        file_path=str(OPHYS_DATA_PATH / "imaging_datasets" / "inscopix" / "movie_longer_than_3_min.isxd")
+    )
+    optical_series_name = "OnePhotonSeries"
+    conversion_options = dict(photon_series_type="OnePhotonSeries")
+
+    def check_extracted_metadata(self, metadata: dict):
+        """Test metadata extraction for file with rich acquisition info."""
+
+        # NWBFile checks
+        nwbfile = metadata["NWBFile"]
+        assert nwbfile["session_start_time"] == datetime(2019, 10, 7, 16, 22, 1, 524186)
+        assert nwbfile["session_id"] == "4D_SAAV_PFC_IM7_20191007"
+
+        # Device checks
+        device = metadata["Ophys"]["Device"][0]
+        assert device["name"] == "NVista3"
+        device_desc = device.get("description", "")
+        if device_desc:
```

Comment:
Let's avoid the if here. If the description is available for this data, remove the if. If it is not, then just remove the assert.

I think this is important so we know exactly what we are testing for. Otherwise, we need to run the code to know that that information is there or know its contents already.